### ISSUE TITLE
[Feature Request] KTO

### ISSUE THREAD

**Issue by jazelly** [mlabonne/llm-course#68]
Hi, this is such an amazing course! Especially all the explanations were making so much sense.

I stumbled my way to KTO training. Here is the [paper](https://arxiv.org/pdf/2402.01306). Can you do an article about that? Looks fanscinating.

**Comment by mlabonne** [comment]
Yeah KTO is super interesting, I didn't have time to seriously explore it yet.
